R/method-differentialAbundance.R

```diff
@@ -52,9 +52,25 @@ setMethod("differentialAbundance", signature("AbsoluteAbundanceData", "Comparato
       veupathUtils::logWithTime("Replaced NAs with 0", verbose)
     }
 
+    # Remove samples with NA from data and metadata
+    if (any(is.na(sampleMetadata[[comparatorColName]]))) {
+      veupathUtils::logWithTime("Found NAs in comparator variable. Removing these samples.", verbose)
+      samplesWithData <- which(!is.na(sampleMetadata[[comparatorColName]]))
```

Comment:
not essential, but id be curious to compare this method against complete.cases for perf.